TAScheduler/tests/AcceptanceTests/test_CreateCourse.py

```diff
@@ -0,0 +1,110 @@
+
+    def test_blankLocation(self):
+        r = self.dummyClient.post('/create_course/', {"ID": 2, "name": "Systems Programming",
+                                                      "location": "", "hours": "12:00PM - 01:00PM",
+                                                      "days": "M, W", "instructor": self.instructor})
+
+        self.assertEqual(r.context["error"],
+                         "Course was not created. invalid literal for int() with base 10: 'Instructor1'",
+                         "An error message was not displayed when name was left blank")
+
+    def test_blankHours(self):
+        r = self.dummyClient.post('/create_course/', {"ID": 2, "name": "Systems Programming",
+                                                      "location": "EMS 180", "hours": "",
+                                                      "days": "M, W", "instructor": self.instructor})
+
+        self.assertEqual(r.context["error"],
+                         "Course was not created. invalid literal for int() with base 10: 'Instructor1'",
+                         "An error message was not displayed when name was left blank")
+
+    def test_invalidHours(self):
+        r = self.dummyClient.post('/create_course/', {"ID": 2, "name": "Systems Programming",
+                                                      "location": "EMS 180", "hours": "invalid input",
+                                                      "days": "M, W", "instructor": self.instructor})
+
+        self.assertEqual(r.context["error"],
+                         "Course was not created. invalid literal for int() with base 10: 'Instructor1'",
+                         "An error message was not displayed when name was left blank")
+
+    def test_blankDays(self):
+        r = self.dummyClient.post('/create_course/', {"ID": 2, "name": "Systems Programming",
+                                                      "location": "EMS 180", "hours": "12:00 PM - 01:00 PM",
+                                                      "days": "", "instructor": self.instructor})
+
+        self.assertEqual(r.context["error"],
+                         "Course was not created. invalid literal for int() with base 10: 'Instructor1'",
+                         "An error message was not displayed when name was left blank")
+
+    def createCourse(self):
+        r = self.dummyClient.post('/create_course/', {"courseID": 99, "name": "Software Engineering",
+                                                      "location": "EMS 180", "hours": "12:00 PM - 01:00 PM",
+                                                      "days": "M, W", "instructor": self.instructor})
+
+        self.assertEqual(Course.objects.get(courseID=1).name, "Software Engineering", "Course was created successfully")
+
+    def createCourseWTA(self):
+        r = self.dummyClient.post('/create_course/', {"courseID": 100, "name": "Systems Programming",
```

Comment:
This courseID differs from the one used below.

Reply:
fixed! Thank you for all the suggestions, acceptance is def not my strong point. I was using those success tests as kind of a base for my other tests, thanks for all the suggestions I committed them. However I am receiving errors saying that they were not created successfully when it should be passing since its comparing 2 equal userProfiles from what I see.

![image](https://user-images.githubusercontent.com/78144364/146738779-c422fb56-aa28-4646-9ba2-27ec1f7c5a9e.png)

Reply:
fixed this issue as well, it was assertNotEquals instead of assertEquals